custom_components/powercalc/data/GE Wink/PSB30-SW27/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly 2 PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.5:docker"
+    },
+    "name": "GE Link BR30 Soft White Bulb",
```

Comment:
```suggestion
    "name": "GE Link PAR38 Bright White Bulb",
```

Reply:
Good catch, thank you. It's making me doubt the validity of either PR now though 🤔

Reply:
@bramstroker Something funky happened and this PR was showing the files for the previous PR. That is fixed now and the name is updated in the json file.

Question - Should the parent directory be `GE Link` instead of `GE Wink`?

Reply:
Looking at the device information you have provided from HA the manufacturer is `GE_Appliances` and the model `ZLL Light`. 
The latter is a generic model, so your Zigbee controller and/or integration does not expose the actual model `PSB38-BW30`. Not sure if and where this can be fixed. Which Zigbee integration do you use?

That being said autodiscovery for this light won't be possible currently.

For the manufacturer name I think just `GE` would be the best fit. So you might rename the directory to `ge`.
Next we also need to add an alias to `custom_components/powercalc/aliases.py`. From `GE_Appliances` to `GE`. That will make discovery work after the model is also correctly exposed from the Zigbee integration. This is not something which can be fixed in Powercalc, but needs to probably be reported at the Zigbee integration issue tracker.

Reply:
> Which Zigbee integration do you use?
ZHA

I'll go ahead and make those changes. Thank you!